src/components/Router/Router.tsx

```diff
@@ -1,9 +1,9 @@
 import React from 'react';
-import './Router.scss';
 import { useRoutes } from 'react-router-dom';
 import { routesConfig } from './routes';
+import './Router.scss';
 
 export const Router = () => {
-  const routes: React.ReactElement | null = useRoutes(routesConfig);
+  const routes: React.ReactElement = useRoutes(routesConfig) || <></>;
```

Comment:
you can use Generic in the hook. Something like this `useRoute<YourOwnType>`

Reply:
using generic is not provided by this hook

![image](https://user-images.githubusercontent.com/77783277/167458917-81d14107-16ea-400b-8c20-0c4e598c3bc4.png)

Reply:
Please use this doc https://reactnavigation.org/docs/typescript/#annotating-useroute

Reply:
we are using react router, not react navigation 

https://reactrouter.com/docs/en/v6/api#useroutes

Reply:
This is just an example how to use typescript

Reply:
The example from the react-router: https://reactrouter.com/docs/en/v6/api#routes-and-route

Reply:
Leave the approach that you had used before.